<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QMainWindow" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>1466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>home</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="loginWidget" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>531</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QLabel" name="loginTitle">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Login girlypop!!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Login :)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="loginTextBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>441</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>25</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="profileWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>1231</width>
      <height>871</height>
     </rect>
    </property>
    <widget class="QLabel" name="profileHeading">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>761</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>35</pointsize>
      </font>
     </property>
     <property name="text">
      <string>My Profile</string>
     </property>
    </widget>
    <widget class="QWidget" name="addFriendWidget" native="true">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>10</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QPushButton" name="addFriendButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tsukushi B Round Gothic</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Friend</string>
      </property>
     </widget>
     <widget class="QLabel" name="addFriendCheck">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>60</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tsukushi B Round Gothic</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="songsWidget" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>791</width>
       <height>531</height>
      </rect>
     </property>
     <widget class="QLabel" name="songsHeading">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>431</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Liked Songs</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="songsTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>771</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QWidget" name="sharedSongsWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>771</width>
        <height>241</height>
       </rect>
      </property>
      <widget class="QLabel" name="sharedSongsHeading">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>741</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Users who like</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="sharedSongsTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>751</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="postsWidget" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>801</width>
       <height>241</height>
      </rect>
     </property>
     <widget class="QLabel" name="postsHeading">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>431</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>My Posts</string>
      </property>
     </widget>
     <widget class="QLabel" name="postsDisplay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>761</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="friendsWidget" native="true">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>100</y>
       <width>381</width>
       <height>441</height>
      </rect>
     </property>
     <widget class="QTableWidget" name="friendsTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>351</width>
        <height>381</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="friendsTitle">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>My Friends</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="suggestedFriendsTable">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>620</y>
       <width>341</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="suggestedFriendsTitle">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>570</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Suggested Friends</string>
     </property>
    </widget>
    <widget class="QWidget" name="addSongWidget" native="true">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>540</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QPushButton" name="addSongButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tsukushi B Round Gothic</family>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Song to my Liked Songs</string>
      </property>
     </widget>
     <widget class="QLabel" name="addSongCheck">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tsukushi B Round Gothic</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>180</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↵</string>
    </property>
   </widget>
   <widget class="QLabel" name="sexyTitle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>691</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>42</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sahana's Super Social Network</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1278</width>
     <height>43</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
